example/4-cascading-zipper/zipper-1/zipper_1.go

```diff
@@ -1,35 +1,41 @@
 package main
 
 import (
+	"context"
+	"fmt"
 	"log"
 	"os"
 
 	"github.com/yomorun/yomo"
+	"github.com/yomorun/yomo/pkg/config"
 )
 
 func main() {
-	zipper, err := yomo.NewZipper("zipper_1_wf.yaml")
+	conf, err := config.ParseWorkflowConfig("zipper_1_wf.yaml")
 	if err != nil {
-		panic(err)
+		log.Fatalln(err)
+	}
+
+	zipper, err := yomo.NewZipper(
+		conf.Name,
+		conf.Functions,
+		yomo.WithDownstreamOption(yomo.WithAuth("token", "z1")),
```

Comment:
'yomo. WithDownstreamOption' confused me.

Reply:
should remove this from `NewZipper()`, we have `AddDownstreamZipper()` to do this.

Reply:
The Zipper has two types: Downstream and Upstream. Use `DownstreamOption` for the Downstream Zipper and `UpstreamOption` for the Upstream Zipper.